_infra/helm/secure-message-v2/templates/deployment.yaml

```diff
@@ -134,5 +134,28 @@ spec:
             {{- end }}
           - name: GOOGLE_APPLICATION_CREDENTIALS
             value: /var/secrets/google/credentials.json
+          - name: UAA_CHECK_ENABLED
+            value: "true"
+          - name: UAA_URL
+            {{- if .Values.dns.enabled }}
+            value: "http://uaa.{{ .Values.namespace }}.svc.cluster.local:{{ .Values.dns.wellKnownPort }}"
+            {{- else }}
+            value: "http://$(UAA_SERVICE_HOST):$(UAA_SERVICE_PORT)"
+            {{- end }}
+          - name: CLIENT_ID
+            valueFrom:
+              secretKeyRef:
+                name: secure-message
```

Comment:
This currently uses the same `secure-message` secret created by the v1 helm chart. When we retire v1 we will need to remember to move that from v1 into this Helm chart. Otherwise the secret won't get created/replicated. There will be some shenanigans involved in doing that gracefully.